src/util/apiPrefix.ts

```diff
@@ -1,5 +1,5 @@
 import isProd from "./isProd";
 
-const apiPrefix = isProd ? "https://cellxgene.cziscience.com" : "/api";
+const apiPrefix = isProd ? "https://cellxgene.cziscience.com" : "http://localhost:5000/api";
```

Comment:
Prod currently needs to read "/api". It should be easy to change "https://cellxgene.cziscience.com" to "/api" and be done with it, but unfortunately, isProd is not currently being read correctly. 

If we don't fix the isProd flag, we have to change this back to: 

`const apiPrefix = isProd ? "https://cellxgene.cziscience.com" : "/api";`

And dev will remain broken for this PR, this is the fastest and is acceptable as this is not the issue being addressed here. If it's obvious and fixable upon investigation, it could also be fixed in this PR.

Reply:
I'm not sure I follow - are you saying that `isProd` is incorrectly set to false in the prod deployment?

Reply:
That is correct — it seems isProd is always false and the ternary always resolves to `/api` https://vitejs.dev/guide/env-and-mode.html